tokio/src/loom/std/parking_lot.rs

```diff
@@ -26,6 +26,12 @@ impl<T> Mutex<T> {
         Mutex(parking_lot::Mutex::new(t))
     }
 
+    #[inline]
+    #[cfg(all(feature = "nightly", feature = "parking_lot", not(all(loom, test)),))]
```

Comment:
let's add
```suggestion
    #[cfg(all(feature = "nightly", feature = "parking_lot", not(all(loom, test)),))]
    #[cfg_attr(docsrs, doc(cfg(all(feature = "parking_lot", feature = "nightly"))))]
```
so that the  required features show up in the RustDoc output?